test/state/state.hpp

```diff
@@ -80,10 +80,11 @@ using AccessList = std::vector<std::pair<address, std::vector<bytes32>>>;
 
 struct Transaction
 {
-    enum class Kind
+    enum class Kind : uint8_t
     {
-        legacy,
-        eip1559
+        legacy = 0,
+        eip2930,  // https://eips.ethereum.org/EIPS/eip-2930
```

Comment:
Add more information, e.g. "transaction with access list" and use doxygen comment `///<`.